spec/helpers/testScheduler-ui.ts

```diff
@@ -0,0 +1,152 @@
+///<reference path='../../typings/main.d.ts'/>
+///<reference path='ambient.d.ts'/>
+
+import * as Rx from '../../dist/cjs/Rx.KitchenSink';
+import * as mocha from 'mocha';
+import * as Suite from 'mocha/lib/suite';
+import * as Test from 'mocha/lib/test';
+import * as commonInterface from 'mocha/lib/interfaces/common';
+import * as escapeRe from 'escape-string-regexp';
+import * as chai from 'chai';
+import * as sinonChai from 'sinon-chai';
+
+//setup sinon-chai
+chai.use(sinonChai);
+
+declare const module: any;
+declare const global: any;
+
+const diagramFunction = global.asDiagram;
+
+//mocha creates own global context per each test suite, simple patching to global won't deliver its context into test cases.
```

Comment:
rxTestscheduler is set up in https://github.com/ReactiveX/rxjs/pull/1545/files#diff-044a0fafe3ba4e4178acb8de9fb82252R90. Actually that's interesting question, what I found of mocha's global context is it doesn't allow patching existing interfaces (i.e, `it`, etcs..) cause its interface constructs it everytime per test suite, as you can see in this implementation. Other global instance seems not being affected. Maybe my word on comment is somewhat misleading that all global contexts are completely ignored..